code/data_structures/src/stack/sort_stack/sort_stack.js

```diff
@@ -0,0 +1,55 @@
+var Stack = function (initStack = [])
```

Comment:
let let let. In fact, use `es6` [classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes).